packages/editor-ui/src/router.ts

```diff
@@ -155,32 +106,6 @@ export const routes = [
 			middleware: ['authenticated'],
 		},
 	},
-	{
-		path: '/templates/',
```

Comment:
should we automatically redirect to n8n.io here?

Reply:
Nice, that way old bookmarks will keep working, thanks!